app/views/permissions/show.html.erb

```diff
@@ -8,7 +10,7 @@
     <p>
       <b>Collections |
         <% if @collection_options %>
-          <%= @collection_entry_ids.blank? ? 'All Collections' : "#{@collection_entry_ids.count} Selected Collections" %>
+          <%= @collection_entry_ids.blank? && @hidden_entry_titles.blank? ? 'All Collections' : "#{@collection_entry_ids.count + @hidden_entry_titles.count} Selected Collections" %>
```

Comment:
Yes, for 2 reasons:
1. The way the logic was working, if the user couldn't see any of the collections (they turned out all be in the hidden field), the show page would say it was a permission for 'All Collections'. We can obviously change it how this works, but it could be weird to have it say '0 Selected Collections'
2. If we have some sort of notification about there being collections the user can't see, then the discrepancy would make sense to them. They might be able to also infer how many collections they can't see, but if that is something we want them to know we should be more direct about it.